### PR TITLE
update badges

### PR DESCRIPTION
**Pull request by SophieHerbst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/mne-tools/mne-study-template.svg?style=svg)](https://circleci.com/gh/mne-tools/mne-study-template)
+[![CircleCI](https://circleci.com/gh/brainthemind/CogBrainDyn_MEG_Pipeline.svg?style=svg)](https://circleci.com/gh/brainthemind/CogBrainDyn_MEG_Pipeline)
```
Comment (agramfort) [inline README.md]: You should not do this. Forks should not change upstream CIs

Comment (SophieHerbst) [reply README.md]: that pull request was a mistake. sorry.

Comment (SophieHerbst) [reply README.md]: how do I undo it?

Comment (agramfort) [reply README.md]: Close pr. That’s it

**Pull request closed without merging**